lib/spack/spack/test/compilers/basics.py

```diff
@@ -806,6 +812,8 @@ def _call(*args, **kwargs):
         assert 'SPACK_TEST_CMP_ON' not in os.environ
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
This seems like a good example for a case where it's easy to improve the `reason`: the test creates a bash script to approximate a compiler, which won't work on Windows.